htlcswitch/switch_test.go

```diff
@@ -3223,8 +3222,8 @@ func TestSwitchHoldForward(t *testing.T) {
 
 	// Create request which should be forwarded from Alice channel link to
 	// bob channel link.
-	preimage := [sha256.Size]byte{1}
-	rhash := fastsha256.Sum256(preimage[:])
+	preimage := lntypes.Preimage{1}
```

Comment:
applied in #4410, tho i'm okay making the switch to use `Hash()` here as well

Reply:
dropped commit